src/components/AccordionItemButton.tsx

```diff
@@ -24,7 +24,11 @@ const AccordionItemButton = ({
     const handleKeyPress = (evt: React.KeyboardEvent<HTMLDivElement>): void => {
         const keyCode = evt.key;
 
-        if (keyCode === keycodes.ENTER || keyCode === keycodes.SPACE) {
+        if (
+            keyCode === keycodes.ENTER ||
+            keyCode === keycodes.SPACE ||
+            keyCode === keycodes.SPACE_DEPCRICATED
```

Comment:
```suggestion
            keyCode === keycodes.SPACE_DEPRECATED
```

Reply:
I'm not saying we should change this as there are only 3 items, but an alternative way of writing this that might be clearer is to use a switch/case for the 3 items, or perhaps a `[keycodes.SPACE_DEPRECATED, keycodes.SPACE, keycodes.ENTER].indexOf(keyCode) !== -1`  ...using the includes array method would be clearer still but IE11 compat lacking